### PR TITLE
Add missing french translations

### PR DESCRIPTION
**Pull request by maximeMD**
### What does it do?

Add all missing french translations in the core/admin package

### CODE REVIEW

File: packages/core/admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -258,7 +258,6 @@
   "app.components.EmptyAttributes.title": "There are no fields yet",
   "app.components.EmptyStateLayout.content-document": "You don't have any content yet...",
   "app.components.EmptyStateLayout.content-permissions": "You don't have the permissions to access that content",
-  "app.components.EmptyStateLayout.content-document": "No content found",
```
Comment (maximeMD) [inline packages/core/admin/admin/src/translations/en.json:261]: duplicate with line 259

**Review by vincentbpro — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just one small change, the rest looks good to me.

**Comment by maximeMD** [comment]
@vincentbpro done ! 😃

**Review by vincentbpro — APPROVED** [review]
LGTM!

**Pull request merged**